core/shared/src/main/scala/org/polyvariant/colorize.scala

```diff
@@ -68,7 +79,12 @@ object colorize {
       newColor: String
     ): ColorizedString = ColorizedString.Overlay(
       underlying = this,
-      prefix = newColor,
+      color = Color.Ansi(newColor),
+    )
+
+    def rgb(red: Int, green: Int, blue: Int): ColorizedString = ColorizedString.Overlay(
+      underlying = this,
+      color = Color.Rgb(red, green, blue),
```

Comment:
RGB color has strict bounds: 0 to 255. A value outside of this range is incorrect and cannot be rendered properly. 

There are several options:
1) Use some kind of a refined type
2) Use assert
3) Do not render incorrect values and log a message
4) Make `rgb` return `Either[, ColorizedString]`

Some options move us towards effects libraries, which would be nice to omit.

Reply:
imo `assert` is the way to go, as long as it's clear from the Scaladoc that values outside of the range will fail.